src/restore.ts

```diff
@@ -26,16 +26,21 @@ async function restore(ccacheVariant : string) : Promise<void> {
 
   core.saveState("primaryKey", primaryKey);
 
-  const restoredWith = await cache.restoreCache(paths, primaryKey, restoreKeys);
-  if (restoredWith) {
-    core.info(`Restored from cache key "${restoredWith}".`);
-    if (SELF_CI) {
-      core.setOutput("test-cache-hit", true)
-    }
+  const shouldRestore = core.getBooleanInput("restore");
+  if (!shouldRestore) {
+    core.info("Restore set to false, skip restoring cache.");
```

Comment:
It would be easier if you `return` after this line to make the code simpler. In addition I would move to the beginning of `restore`.

Reply:
I originally had it earlier on in the code, but save requires `core.saveState("primaryKey", primaryKey);` so I had to move it later on into the code. As far as a return you think just a generic `return 0;`?

Reply:
:+1: didn't see that, good point. 

> As far as a return you think just a generic return 0;?

`return;` without any value should work.